sqlmesh/core/console.py

```diff
@@ -1036,7 +1036,7 @@ def show_schema_diff(self, schema_diff: SchemaDiff) -> None:
         if schema_diff.target_alias:
             target_name = schema_diff.target_alias.upper()
 
-        first_line = f"\n[b]Schema Diff Between '[yellow]{source_name}[/yellow]' and '[green]{target_name}[/green]'"
+        first_line = f"\n[b]Schema Diff Between '[#0248ff]{source_name}[/#0248ff]' and '[green]{target_name}[/green]'"
```

Comment:
I'd consider factoring out the hex code for the color into a global constant, e.g. `TABLE_DIFF_SOURCE_YELLOW = ...` or something - right now it's unintelligible.